Tests/RoutingTests/RouterTests.swift

```diff
@@ -117,6 +132,8 @@ class RouterTests: XCTestCase {
 }
 
 final class User: Parameter {
+    typealias ParameterType = String
```

Comment:
I'm thinking we can add extensions to Fluent types to have `ParameterType = Model.ID` by default. That way this PR isn't realistically likely to require any extra work by Vapor users.